src/main.rs

```diff
@@ -36,8 +36,13 @@ enum CommandOpts {
 
 #[derive(StructOpt)]
```

Comment:
it appears that `structopt` is being deprecated in favor of `clap`: https://docs.rs/structopt/latest/structopt/

Reply:
see https://github.com/ScaleComputing/cpuinfo/issues/12